frontend/packages/console-shared/src/components/dynamic-form/fields.tsx

```diff
@@ -121,15 +112,10 @@ export const UpdateStrategyField: React.FC<FieldProps> = (props) => {
   );
 };
 
-export const NodeAffinityField: React.FC<FieldProps> = ({
-  formData,
-  idSchema,
-  name,
-  onChange,
-  required,
-}) => {
+export const NodeAffinityField: React.FC<FieldProps> = (props) => {
+  const { formData, idSchema, onChange } = props;
   return (
-    <FieldSet title={name} idSchema={idSchema} required={required}>
+    <FieldSet {...props}>
```

Comment:
If I recall, its generally preferred to avoid the `{...props}` spread on an elem, no?

Reply:
Yeah, if you prefer I can update this and break these down to individual props.